src/feature/flow_chart_panel/views/NodeModal.tsx

```diff
@@ -17,9 +19,36 @@ const NodeModal = ({
   clickedElement,
 }: NodeModalProps) => {
   const theme = useMantineTheme();
-
+  const GLINK = "https://github.com/flojoy-io/nodes/blob/main";
+  let nodeCategory = "";
+  let nodeTypeLink = "";
+  const nodeDataLabel = pythonString.split(".")[0];
   const colorScheme = theme.colorScheme;
 
+  CMND_TREE.child?.forEach((outer) => {
+    outer.child?.forEach((inner) => {
+      if (inner.key === nodeType) {
+        nodeCategory = outer.title.toUpperCase();
+        nodeTypeLink = inner.title.toUpperCase();
+        nodeCategory = nodeCategory.replace(/\s/g, "_");
+        nodeTypeLink = nodeTypeLink.replace(/\s/g, "_");
+        if (nodeTypeLink === "CLOUD_DATABASES") nodeTypeLink = "CLOUD_DATABASE";
+      }
+    });
+  });
+
+  let LINK = "";
+  switch (nodeDataLabel) {
+    case "END":
+      LINK = `${GLINK}/${nodeCategory}/${nodeTypeLink}/${pythonString}`;
+      break;
+    case "OBJECT_DETECTION":
+      LINK = `${GLINK}/AI_ML/${nodeDataLabel}/${pythonString}`;
```

Comment:
We should probably not hardcode "AL_ML" here I think.

Reply:
Oh okay! The file name was acquired using the title property from COMMANDS_MANIFEST.ts file and the title name was something like "AI and Machine Learning". What alternative do you suggest in this situation?
![Screenshot 2023-05-23 at 2 59 16 PM](https://github.com/flojoy-io/studio/assets/83793112/fcb1a75f-8c71-4bd0-9dc8-089599f53533)

Reply:
I agree with hard coding being a malpractice here.

Reply:
@smahmed776 Shall we just change the title to stay consistent in this case?

Reply:
@itsjoeoui Similar thing for CLOUD_DATABASE

Reply:
@itsjoeoui  We can add a key field for each parent category and update all folders not matching the key(if any) and keep the title as it is because it looks better with spacing and full name in the sidebar than these shorthands IMO. Also, we need to ensure each node folder follows its file name like `LINSPACE/LINSPACE.py`. This should resolve all the issues.

note: We need to update docs in the docs repo to force app builders to follow this convention.

Reply:
Sounds good! I think we can start with that such that we can have a better structure, I can take a look in a bit

Reply:
Sounds good! I'll be working on it today!